README_STAC_MODEL.md

```diff
@@ -65,6 +65,62 @@ with open("examples/mlm-metadata.yaml", "r", encoding="utf-8") as f:
 MLModelProperties.model_validate(metadata["properties"])  
 ```
 
+## Exporting and Packaging PyTorch Models, Transforms, and Model Metadata
+
+As of PyTorch 2.8, and stac_model 1.5.0, you can now export and package PyTorch models, transforms,
```

Comment:
```suggestion
As of PyTorch 2.8, and stac_model 0.4.0, you can now export and package PyTorch models, transforms,
```

the stac-model version is incremented differently than the mlm schema version.

https://pypi.org/project/stac-model/